src/backend/vulkan/src/lib.rs

```diff
@@ -511,6 +511,11 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
     }
 
     fn features(&self) -> Features {
+        // see https://github.com/gfx-rs/gfx/issues/1930
+        let is_windows_intel_kaby = cfg!(windows) &&
+            self.properties.vendor_id == 0x8086 &&
+            self.properties.device_id >> 2 == 0x59;
```

Comment:
Do you know that this only applies Kabylake?

Reply:
Or to state more clearly, are there other Intel GPUs where this might be broken?

Reply:
I based the filter on all the related issues of #1930 (reported in other places on the net). There might be more cases, we'll add them later anyway.